build.js

```diff
@@ -0,0 +1,87 @@
+import fs from 'fs';
+
+import copy from 'esbuild-plugin-copy';
+import esbuild from "esbuild";
+import { sassPlugin } from 'esbuild-sass-plugin';
+
+import { cockpitCompressPlugin } from './pkg/lib/esbuild-compress-plugin.js';
+import { cockpitPoEsbuildPlugin } from './pkg/lib/cockpit-po-plugin.js';
+import { cockpitRsyncEsbuildPlugin } from './pkg/lib/cockpit-rsync-plugin.js';
+import { eslintPlugin } from './pkg/lib/esbuild-eslint-plugin.js';
+import { stylelintPlugin } from './pkg/lib/esbuild-stylelint-plugin.js';
+
+const production = process.env.NODE_ENV === 'production';
+const watchMode = process.env.ESBUILD_WATCH === "true" || false;
+// linters dominate the build time, so disable them for production builds by default, but enable in watch mode
+const lint = process.env.LINT ? (process.env.LINT !== '0') : (watchMode || !production);
+/* List of directories to use when resolving import statements */
+const nodePaths = ['pkg/lib'];
+const outdir = 'dist';
+
+// always start with a fresh dist/ directory, to change between development and production, or clean up gzipped files
+const cleanPlugin = {
```

Comment:
I suppose this should also eventually move to cockpit's pkg/lib/, but at this point my gut feeling is that we can land this PR, and do that separately. @KKoukiou is already working on esbuild-ifying cockpit.git, and she will most certainly add the clean plugin there as well.